components/list/list-item-drag-mixin.js

```diff
@@ -1,20 +1,77 @@
+import { css, html } from 'lit-element/lit-element.js';
 import { announce } from '../../helpers/announce.js';
+import { dragActions } from './list-item-drag-handle.js';
 import { getUniqueId } from '../../helpers/uniqueId.js';
-import { html } from 'lit-element/lit-element.js';
 import { nothing } from 'lit-html';
 
+const move = Object.freeze({
+	above: true,
+	below: false
+});
+
+const timeDelayForLeavingTheDropArea = 1000; //ms
+
 export const ListItemDragMixin = superclass => class extends superclass {
 
 	static get properties() {
 		return {
 			draggable: { type: Boolean, reflect: true },
-			key: { type: String, reflect: true }
+			dragText: { type: String, attribute: 'drag-text' },
```

Comment:
If someone drags a course to the URL bar or a notepad. This is what will be dropped. Gives some neat functionality with no cost so why not.